### DISCUSSION TITLE
I tried to drag and drop this....

### DISCUSSION THREAD

**Discussion by therriend** [Ghadjeres/piano-inpainting-application#6] [Q&A] [answered]
Hi I downloaded the amxd file and tried to drag and drop it into MIDI Effects or Max for Live MIDI effects and it wouldn't copy. I was able to add it as a MIDI effect in the Device area. but I'd like to save it somewhere to use again. I am running the latest version of Ableton Live 11 Suite

**Comment by Ghadjeres** [comment, marked as answer]
Hi Dave,
Thanks a lot for you interest.
I just did the following thing:
- Opened Ableton Live 11 (in trial version mode),
- Downloaded the amxd from https://github.com/Ghadjeres/piano-inpainting-application/releases/download/v0.1-alpha/PIA.amxd
- Dragged the file from my desktop to the MIDI effect zone as you can see in the beginning of https://www.youtube.com/watch?v=YKCzU8UYsRU&t=4s (the only difference is that I dragged it from my explorer, instead of dragging it from the user library from Ableton __ but you can also first configure your Ableton User Library (right click on it) and then drag the PIA plugin in it to reuse it later__ .
- Selected and looped a region of the midi clip and clicked on the PIA button (you'll then be prompted with a compatibility window if you're on Ableton 11, but it works)
- And you should see notes appearing! You can now use your favorite virtual instrument.
Hope this helps and that it will work.

**Comment by therriend** [comment]
Hi 

I think the mistake that I was making is that I tried to drag it over MIDI Effects in the Categories section. I needed to store it in my user library, which makes perfect sense now!

Thanks - I’ll play around with this later today.. 

Dave

**Comment by NOURMUSIC-CREAT** [comment]
Hello, sir.
I'm my name is Nour Belarbi . A Magseter student is interested in all things technology, music and especially coding.
Can you help me with my research because I'm going to study your project, It is an analytical study of Deep Bach ?
and THANKS!
